apps/accounts/verify.py

```diff
@@ -9,17 +11,27 @@
 def fxa_identify(code, config=None):
     """Get an FxA profile for an access token. If identification fails an
     IdentificationError is raised."""
-    token = get_fxa_token(code, config)['access_token']
-    return get_fxa_profile(token, config)
+    statsd.incr('accounts.fxa.identify.count.all')
+    try:
+        with statsd.timer('accounts.fxa.identify.time.all'):
```

Comment:
you don't need `time` in here because timers are already namespaced separately from counts